.github/workflows/ci.yml

```diff
@@ -52,6 +52,7 @@ jobs:
 
   tests:
     runs-on: ubuntu-latest
+    timeout-minutes: 5
```

Comment:
i had an issue with freezegun and async tests so added this to prevent ci from just running forever when the tests hang